requirements/edx/base.in

```diff
@@ -75,7 +75,7 @@ edx-django-release-util             # Release utils for the edx release pipeline
 edx-django-sites-extensions==2.3.1
 edx-django-utils
 edx-drf-extensions
-edx-enterprise
+edx-enterprise==1.6.9
```

Comment:
Why is this 1.6.9 when the rest is 1.6.7?

Reply:
Good catch, not sure why that wasn't updated with https://github.com/edx/edx-platform/commit/375ccb7bf92c85c8626a9196b71d8fa6a6e20f58

Fixed now. Once we've resolved all the issues we can go back to unpinning this...